src/NServiceBus.AmazonSQS.AcceptanceTests/SqsTransportConfigurationExtensions.cs

```diff
@@ -4,11 +4,6 @@
 
     public static class SqsTransportConfigurationExtensions
     {
-
```

Comment:
I had to manually implemented the changes from hotfix 4.1.2 here and figured I also apply the reformat